### PR TITLE
ci: Push strings to Crowdin on string changes

### PR DESCRIPTION
**Pull request by Ushie**

### CODE REVIEW

File: .github/workflows/pull_strings.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Pull strings from Crowdin
```
Comment (oSumAtrIX) [inline .github/workflows/pull_strings.yml:1]:
```suggestion
name: Pull strings
```

File: .github/workflows/push_strings.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name: Push strings to Crowdin
```
Comment (oSumAtrIX) [inline .github/workflows/push_strings.yml:1]:
```suggestion
name: Push strings
```

File: .github/workflows/push_strings.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+on:
+    workflow_dispatch:
+    push:
+        paths:
+            - /src/main/resources/addresources/values/strings.xml
+
+jobs:
+    sync:
+        name: Sync
+        runs-on: ubuntu-latest
```
Comment (oSumAtrIX) [inline .github/workflows/push_strings.yml:12]:
```suggestion
    push:
        name: Push strings
        runs-on: ubuntu-latest
```

File: .github/workflows/pull_strings.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+on:
+    workflow_dispatch:
+    schedule:
+        - cron: 0 * 1 * *
+
+jobs:
+    sync:
+        name: Sync
+        runs-on: ubuntu-latest
```
Comment (oSumAtrIX) [inline .github/workflows/pull_strings.yml:11]:
```suggestion
    pull:
        name: Pull strings
        runs-on: ubuntu-latest
```

File: .github/workflows/push_strings.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+              with:
+                  fetch-depth: 0
+
+            - name: Push strings to Crowdin
```
Comment (oSumAtrIX) [inline .github/workflows/push_strings.yml:19]:
```suggestion
            - name: Push strings
```

File: .github/workflows/push_strings.yml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+              uses: crowdin/github-action@v1
+              with:
+                  config: crowdin.yml
+                  upload_sources: true
+                  upload_translations: false
+                  download_translations: false
```
Comment (oSumAtrIX) [inline .github/workflows/push_strings.yml:25]:
```suggestion
```

File: .github/workflows/pull_strings.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+              uses: actions/checkout@v4
+              with:
+                  fetch-depth: 0
+
+            - name: Pull strings from Crowdin
+              uses: crowdin/github-action@v1
+              with:
+                  config: crowdin.yml
+                  upload_sources: false
+                  upload_translations: false
```
Comment (oSumAtrIX) [inline .github/workflows/pull_strings.yml:23]:
```suggestion
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: .github/workflows/pull_strings.yml [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    pull:
+        name: Pull strings
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+              with:
+                  fetch-depth: 0
+
+            - name: Pull strings from Crowdin
```
Comment (oSumAtrIX) [inline .github/workflows/pull_strings.yml:18]:
```suggestion
            - name: Pull strings
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: .github/workflows/pull_strings.yml [hunk 9]
```diff
@@ -0,0 +1,34 @@
+name: Pull strings
+
+on:
+    workflow_dispatch:
+    schedule:
+        - cron: 0 * 1 * *
```
[the remark this answers is not in the document]
Comment (Ushie) [reply .github/workflows/pull_strings.yml:6]: Oops, no, it's not 🤦‍♂️

@oSumAtrIX 4 eyes again

**Pull request merged**